webapp/publisher/snaps/views.py

```diff
@@ -806,6 +806,33 @@ def post_register_name():
     return flask.redirect(flask.url_for("account.get_account"))
 
 
+@publisher_snaps.route("/register-snap-json", methods=["POST"])
```

Comment:
@jpmartinspt @tbille I'm not sure about the URL. If we want a JSON version endpoint of a route that we already have (/register-name) what would be good url? `register-name-json`, `register-name/json`?

Or maybe it could be on the same route/URL and just respond with JSON based on headers?

Reply:
I prefer the second name `register-name/json`